# HG changeset patch
# User Thorsten Liebig <Thorsten.Liebig@gmx.de>
# Date 1346857173 -7200
#      Wed Sep 05 16:59:33 2012 +0200
# Node ID 34ae358bd5663c380b86040554e42f6f410c3b15
# Parent  d811058962886e36c8b8568bf6c369f124980898
* contributors.in: restored alphabetic order and removed a dublicate entry

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -92,16 +92,17 @@ Dave Hawthorne
 Daniel Heiserer
 Martin Helm
 Stefan Hepp
 Jordi Gutiérrez Hermoso
 Yozo Hida
 Ryan Hinton
 Roman Hodek
 A. Scottedward Hodel
+Júlio Hoffimann
 Richard Allan Holcombe
 Tom Holroyd
 David Hoover
 Kurt Hornik
 Christopher Hulbert
 Cyril Humbert
 John Hunt
 Teemu Ikonen
@@ -184,16 +185,17 @@ Rick Niles
 Takuji Nishimura
 Kai Noda
 Eric Norum
 Krzesimir Nowak
 Michael O'Brien
 Peter O'Gorman
 Thorsten Ohl
 Arno Onken
+Valentin Ortega-Clavero
 Luis F. Ortiz
 Scott Pakin
 Gabriele Pannocchia
 Sylvain Pelissier
 Per Persson
 Primozz Peterlin
 Jim Peterson
 Danilo Piazzalunga
@@ -290,11 +292,8 @@ Andreas Weingessel
 Martin Weiser
 Michael Weitzel
 David Wells
 Fook Fah Yap
 Sean Young
 Michael Zeising
 Federico Zenith
 Alex Zvoleff
-Valentin Ortega-Clavero
-Jacob Dawid
-Júlio Hoffimann
